src/pymor/models/iosys.py

```diff
@@ -522,21 +548,7 @@ def __mul__(self, other):
         return self.with_(A=A, B=B, C=C, D=D, E=E)
 
     @cached
-    def poles(self, mu=None):
-        """Compute system poles.
-
-        .. note::
-            Assumes the systems is small enough to use a dense eigenvalue solver.
-
-        Parameters
-        ----------
-        mu
-            |Parameter values| for which to compute the systems poles.
-
-        Returns
-        -------
-        One-dimensional |NumPy array| of system poles.
-        """
+    def _poles(self, mu=None):
         if not isinstance(mu, Mu):
             mu = self.parameters.parse(mu)
         assert self.parameters.assert_compatible(mu)
```

Comment:
It would be good to move this part to the `poles` function (to avoid potentially caching both parsed and unparsed `mu`).